src/components/publicApi/types.ts

```diff
@@ -1 +1,24 @@
 export type InitialRouteName = 'login' | 'login-password' | 'new-account'
+
+/**
+ * Subset of AppConfig from edge-react-gui
```

Comment:
A bit more explicit and in-your-face for the reviewer to catch in the future mutations to this type.

```
Subset of AppConfig from edge-react-gui (<link to file>).
This MUST always maintain as a subset
```